### PR TITLE
tr(CI): integrate GH Actions build badge

### PR DESCRIPTION
**Pull request by educhastenier**
on README.md file
as a result of the build status

### CODE REVIEW

**Comment by educhastenier** [comment]
Will redo it on SP repository

File: README.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
-# Bonita Engine [![Build Status](https://travis-ci.org/bonitasoft/bonita-engine.svg)](https://travis-ci.org/bonitasoft/bonita-engine)
+# Bonita Engine
 
+[![Travis-ci Status](https://travis-ci.org/bonitasoft/bonita-engine.svg)](https://travis-ci.org/bonitasoft/bonita-engine)
```
Comment (tbouffard) [inline README.md]:
```suggestion
[![Travis-ci Status](https://travis-ci.org/bonitasoft/bonita-engine.svg)](https://travis-ci.org/bonitasoft/bonita-engine)
![Travis CI](https://img.shields.io/travis/bonitasoft/bonita-engine/master?label=Travis%20build&logo=travis)
```

Comment (tbouffard) [reply README.md]: the following suggestion add a travis logo

image

Comment (educhastenier) [reply README.md]: ok

File: README.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
...
+
+[![Github Actions status](https://github.com/bonitasoft/bonita-engine/workflows/Build%20Engine%20Community/badge.svg)](https://github.com/bonitasoft/bonita-engine/actions)
```
[the remark this answers is not in the document]
Comment (educhastenier) [reply README.md]: ok

**Pull request closed without merging**